mmv1/third_party/terraform/tests/resource_cloudfunctions_function_test.go.erb

```diff
@@ -918,3 +1002,152 @@ resource "google_cloudfunctions_function" "function" {
 }
 `, projectNumber, networkName, vpcConnectorName, vpcConnectorName, vpcIp, bucketName, zipFilePath, functionName, vpcConnectorName)
 }
+
+func testAccCloudFunctionsFunction_secretEnvVar(projectNumber, secretName, versionName, bucketName, functionName, versionNumber, zipFilePath string) string {
+	return fmt.Sprintf(`
+data "google_project" "project" {}
+
+resource "google_project_iam_member" "gcfadmin" {
+  project = data.google_project.project.project_id
+  role     = "roles/editor"
+  member   = "serviceAccount:service-%s@gcf-admin-robot.iam.gserviceaccount.com"
+}
+
+resource "google_service_account" "cloud_function_runner" {
+  account_id   = "cloud-function-service"
```

Comment:
Can we use a `tf-test-%{random_suffix}` name here? Tests are failing due to repeated names

Reply:
Done.